DEBUG_template.toml

```diff
@@ -0,0 +1,49 @@
+[LIBRARY]
```

Comment:
Could you please add a note here at the very top, `# customize this file and save as DEBUG.toml`?

Reply:
done